data/mods/ssb/abilities.ts

```diff
@@ -1504,10 +1534,10 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 	},
 	snowcloak: {
 		inherit: true,
-		desc: "If Heavy Hailstorm or Hail is active, this Pokemon's evasiveness is multiplied by 1.25. This Pokemon takes no damage from Heavy Hailstorm or Hail.",
+		desc: "If Heavy Hailstorm, Winter Hail, or Hail is active, this Pokemon's evasiveness is multiplied by 1.25. This Pokemon takes no damage from Heavy Hailstorm or Hail.",
 		shortDesc: "If a Hail-like weather is active, 1.25x evasion; immunity to Hail-like weathers.",
 		onImmunity(type, pokemon) {
-			if (['heavyhailstorm', 'hail'].includes(type)) return false;
+			if (['heavyhailstorm', 'hail', 'winterhail'].includes(type)) return false;
```

Comment:
Correct me if this is wrong, but it seemed to be an oversight to me.

Reply:
probably is